### DISCUSSION TITLE
How to use your project name

### DISCUSSION THREAD

**Discussion by Ambassador-of-programming** [flet-dev/flet#2636] [Q&A]
### Question

How can I use my own project name when creating an apk file for example?
Basically when we use `flet build apk` the name is taken from the name of our folder and with small letters.
How can I use my own name?

### Code sample

```python
flet build apk
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
```bash
flet build apk --product "App Name"
```

**Reply by carlosjunior-br** [reply]
O parâmetro --product não aceita nome composto (com espaços) nas versões mais recentes do flet. Existe alguma forma atual de definir o nome do aplicativo?

**Reply by ndonkoHenri** [reply]
Can you provide an example?

**Reply by carlosjunior-br** [reply]
Desisti do flet. Migrei para html, javascript com cordova e tudo funcionando bem.
